support/jaxrs/src/main/java/org/apache/shiro/web/jaxrs/ShiroFeature.java

```diff
@@ -55,9 +53,7 @@ public class ShiroFeature implements Feature {
 
     @Override
     public boolean configure(FeatureContext context) {
-
         context.register(UnauthorizedExceptionExceptionMapper.class);
-        context.register(UnauthenticatedException.class);
```

Comment:
UnauthenticatedException should be replaced with UnauthenticatedExceptionExceptionMapper.class.

It is already inside the jar

Reply:
That is already there, just prior to the line you have here

Reply:
NVM I see...